docs/sources/setup-grafana/configure-security/configure-authentication/okta/index.md

```diff
@@ -24,6 +24,85 @@ If Users use the same email address in Okta that they use with other authenticat
 
 To follow this guide, ensure you have permissions in your Okta workspace to create an OIDC app.
 
+## Create an Okta app
+
+1. From the Okta Admin Console, select **Create App Integration** from the **Applications** menu.
+1. For **Sign-in method**, select **OIDC - OpenID Connect**.
+1. For **Application type**, select **Web Application** and click **Next**.
+1. Configure **New Web App Integration Operations**:
+
+   - **App integration name**: Choose a name for the app.
+   - **Logo (optional)**: Add a logo.
+   - **Grant type**: Select **Authorization Code** and **Refresh Token**.
+   - **Sign-in redirect URIs**: Replace the default setting with the Grafana Cloud Okta path, replacing <YOUR_ORG> with the name of your Grafana organization: https://<YOUR_ORG>.grafana.net/login/okta. For on-premises installation, use the Grafana server URL: http://<my_grafana_server_name_or_ip>:<grafana_server_port>/login/okta.
+   - **Sign-out redirect URIs (optional)**: Replace the default setting with the Grafana Cloud Okta path, replacing <YOUR_ORG> with the name of your Grafana organization: https://<YOUR_ORG>.grafana.net/login/okta. For on-premises installation, use the Grafana server URL: http://<my_grafana_server_name_or_ip>:<grafana_server_port>/login/okta.
+   - **Base URIs (optional)**: Add any base URIs
+   - **Controlled access**: Select whether to assign the app integration to everyone in your organization, or only selected groups. You can assign this option after you create the app.
+
+1. Make a note of the following:
+   - **ClientID**
+   - **Client Secret**
+   - **Auth URL**
+     For example: https://<TENANT_ID>.okta.com/oauth2/v1/authorize
+   - **Token URL**
+     For example: https://<TENANT_ID>.okta.com/oauth2/v1/token
+   - **API URL**
+     For example: https://<TENANT_ID>.okta.com/oauth2/v1/userinfo
+
+### Configure Okta to Grafana Cloud role mapping
+
+1. In the **Okta Admin Console**, select **Directory > Profile Editor**.
+1. Select the Okta Application Profile you created previously (the default name for this is `<App name> User`).
+1. Select **Add Attribute** and fill in the following fields:
+
+   - **Data Type**: string
+   - **Display Name**: Meaningful name. For example, `Grafana Role`.
+   - **Variable Name**: Meaningful name. For example, `grafana_role`.
+   - **Description (optional)**: A description of the role.
+   - **Enum**: Select **Define enumerated list of values** and add the following:
+     - Display Name: Admin Value: Admin
+     - Display Name: Editor Value: Editor
+     - Display Name: Viewer Value: Viewer
+
+   The remaining attributes are optional and can be set as needed.
```

Comment:
I used a Personal Attribute Type in my reproduction, though I believe the customer we had a call with were using Group types. I'm not entirely sure it makes a difference but thought it should be pointed out.

Reply:
Yes, there's a small difference, but it depends on the particular Okta setup, that's why I added the `can be set as needed`.